matico_components/src/Stores/MaticoSpecSlice.tsx

```diff
@@ -15,6 +15,51 @@ const initialState: SpecState = {
   editing: false,
 };
 
+const EditTypeMapping = {
+  'pages': 'Page',
+  'layers': 'Layer',
+  'Map': 'Map',
+  'sections': 'Section',
+  'Text': 'Text',
+  'Histogram':'Histogram',
+  'PieChart':'PieChart',
+  'Scatterplot':'Scatterplot',
+  'Controls':'Controls'
+}
+
+const extractEditType = (path: string): string => {
```

Comment:
These are all super useful util functions thanks!

Reply:
Great! Can pull these over to utils in a cleanup/refactor before merging